src/build/analyzer.ts

```diff
@@ -241,33 +244,20 @@ class StreamResolver implements Resolver {
   }
 
   accept(url: string, deferred: Deferred<string>): boolean {
-    let parsed = parseUrl(url);
-    let filepath: string;
+    let urlObject = parseUrl(url);
 
-    if (!parsed.hostname) {
-      filepath = parsed.pathname;
+    if (urlObject.hostname || !urlObject.pathname) {
+      return false;
     }
 
-    // this.analyzer.requestedUrls.add(local);
+    let filepath = decodeURIComponent(urlObject.pathname);
+    let file = this.analyzer.files.get(filepath);
 
-    if (filepath) {
-      // un-escape HTML escapes
-      filepath = decodeURIComponent(filepath);
-
-      // If the file path is not already under root, such as /bower_components/...,
-      // prefix it with root
-      if (!filepath.startsWith(this.analyzer.root)) {
-        filepath = path.join(this.analyzer.root, filepath);
-      }
-
-      let file = this.analyzer.files.get(filepath);
-      if (file) {
-        deferred.resolve(file.contents.toString());
-      } else {
-        logger.info('No file found for', filepath);
-      }
-      return true;
+    if (file) {
+      deferred.resolve(file.contents.toString());
+    } else {
+      throw new Error(`No file found for ${filepath}`);
```

Comment:
I did that and it ended up being wrong because Hydrolysis actually rejects... I left the line in commented out with a comment not to add it back :)